src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function sum(x) {
+    count++;
+
+    return function(y) {
+      if (count % 2 === 0 && count > 3) {
+        return 'Bzzz... Error!';
+      }
```

Comment:
You need to update the counter when you are 100% doing something to avoid any unexpected results.
```suggestion
  function sum(x) {
    return function(y) {
      count++;
      
      if (count % 2 === 0 && count > 3) {
        return 'Bzzz... Error!';
      }
     
```